### PR TITLE
Clarify that Element Proxy objects can only be created by user agents

### PR DESCRIPTION
**Pull request by stephenmcgruer**
It is no longer possible in the current version of the spec for user
script to create element proxy objects (nor are they posted to the
animator instance). Instead, the user agent handles creating them and
passing them into the animate() call.

### CODE REVIEW

File: Overview.bs [hunk 1]
```diff
@@ -338,42 +338,29 @@ the user agent <em>must</em> run the following steps:
...
+values, and a map of proxied output properties to their values. The input and output properties in
+the maps are those specified in <a>registerAnimator</a> for the given <a>animator instance</a>.
 
 <pre class='idl'>
 [
-    Exposed=(Window,AnimationWorklet),
-    RaisesException=Constructor,
-    Constructor(Animator animator, Element element, sequence<DOMString> proxiedAttributes)
+    Exposed=(AnimationWorklet),
+    NoInterfaceObject,
```
Comment (flackr) [inline Overview.bs]: Using NoInterfaceObject seems to be advised against in general. Let's allow the interface to be exposed for now and can have an issue about whether to add NoInterfaceObject.

Comment (stephenmcgruer) [reply Overview.bs]: Done.

File: Overview.bs [hunk 2]
```diff
@@ -279,41 +279,30 @@ the user agent <em>must</em> run the following steps:
...
-
-    2. If |element| is null or not an instance of {{Element}} abort the following steps.
-
-    4. <a>Create an element proxy</a> with |element|.
-
-Issue: Todo: Remove this as we no longer allow direct construction of ElementProxy. 
-
-
-When user agent wants to <dfn>create an element proxy</dfn> given |element|, it
+When user agent wants to <dfn>create an element proxy</dfn> given |animatorObj| and |element|, it
```
Comment (flackr) [inline Overview.bs]: What is animatorObj? Is this not the same thing as animatorInstance? I think a clarification here would be creating an element proxy for a given |animatorInstance| and |element|, and then we can remove step 1, which seems ambiguous currently as there may be more than one animator instances associated with a single |element|.

**Review by flackr — CHANGES_REQUESTED** [review]

File: Overview.bs (hunk 2 above)
Comment (stephenmcgruer) [reply Overview.bs]: That's a merge conflict bug. This CL did not touch that line, however the reference to 'animatorObj' was originally removed in https://github.com/WICG/animation-worklet/commit/3066833d29303cad4eac5bca45c3bf00f827a753

It looks like you didn't do the rebase of this CL right, so it ended up re-adding it?

**Review by flackr — APPROVED** [review]

**Pull request merged**